### PR TITLE
Feature/condo/doma 4085/import contacts roles

### PR DESCRIPTION
**Pull request by AleX83Xpert**
Ability to set contact role while import contacts from excel file

### CODE REVIEW

File: apps/condo/domains/contact/hooks/useImporterFunctions.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+import { searchContacts, searchProperty } from '@condo/domains/ticket/utils/clientSchema/search'
 import { useApolloClient } from '@condo/next/apollo'
-import { useAddressApi } from '@condo/domains/common/components/AddressApi'
-import get from 'lodash/get'
-import { Contact } from '../utils/clientSchema'
-import { searchProperty, searchContacts } from '@condo/domains/ticket/utils/clientSchema/search'
 import { useIntl } from '@condo/next/intl'
-import { FLAT_UNIT_TYPE, APARTMENT_UNIT_TYPE, WAREHOUSE_UNIT_TYPE, PARKING_UNIT_TYPE, COMMERCIAL_UNIT_TYPE } from '@condo/domains/property/constants/common'
+import { useOrganization } from '@condo/next/organization'
+import get from 'lodash/get'
```
Comment (toplenboren) [inline apps/condo/domains/contact/hooks/useImporterFunctions.tsx:15]: Best to reorder imports: 

1. external imports (lodash, next)
2. internal imports from `@packages` 
3. internal imports from `@common`
4. other imports

**Review by Alllex202 — APPROVED** [review]

File: apps/condo/domains/contact/hooks/useImporterFunctions.tsx (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/contact/hooks/useImporterFunctions.tsx:15]: Imports sorted alphabetically `ctrl` + `alt` + `o`

File: apps/condo/domains/contact/hooks/useImporterFunctions.tsx [hunk 2]
```diff
@@ -55,19 +64,47 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
 
     const contactCreateAction = Contact.useCreate({})
 
+    const {
+        loading: isRolesLoading,
+        objs: roles,
```
Comment (sitozzz) [inline apps/condo/domains/contact/hooks/useImporterFunctions.tsx:69]:
```suggestion
        objs: contactRoles,
```

Comment (AleX83Xpert) [reply apps/condo/domains/contact/hooks/useImporterFunctions.tsx]: Fixed

**Review by sitozzz — APPROVED** [review]

**Pull request merged**